src/main/java/org/opentripplanner/netex/NetexModule.java

```diff
@@ -55,10 +55,11 @@ public void buildGraph(
     HashMap<Class<?>, Object> extra,
     DataImportIssueStore issueStore
   ) {
-    transitModel.clearTimeZone();
-    CalendarServiceData calendarServiceData = transitModel.getCalendarDataService();
-    boolean hasTransit = false;
     try {
+      transitModel.clearTimeZone();
+      var calendarServiceData = new CalendarServiceData();
+      boolean hasTransit = false;
```

Comment:
I was wondering if would be possible to infer `hasTransit` with a very simple check if there are any agencies, like `agencies.isEmpty()`. That would save us the setter. WDYT?

Reply:
Good point, the business rules for what `hasTransit` is, should be encapsulated inside the model. It is a bit more tricky than checking for agencies, since the definition is bound to active services, but I will at least push the logic inside the TransitModel.

Reply:
We want to get rid of the CalendarService, so I think I will leave part of the to that refactoring, and only move the logic inside the TransitModel. It is possible that we can implement the `hasTransit()` to check for trips, but that would be a functonal change - which I do not want to include in a refactoring - introducing risk.